src/libraries/Microsoft.PowerFx.Interpreter/Functions/FileInfoImpl.cs

```diff
@@ -6,15 +6,18 @@
 using System.Threading.Tasks;
 using Microsoft.PowerFx.Core.Functions;
 using Microsoft.PowerFx.Core.Texl.Builtins;
+using Microsoft.PowerFx.Functions;
 using Microsoft.PowerFx.Interpreter;
 using Microsoft.PowerFx.Types;
 
 namespace Microsoft.PowerFx
 {
-    internal class FileInfoFunctionImpl : FileInfoFunction, IAsyncTexlFunction3
```

Comment:
>IAsyncTexlFunction3 [](http://example.com/codeflow?start=60&length=19)

This is just demonstrating an example of converting IAsyncTexlFunction3.  The other uses look similarly easy.